### DISCUSSION TITLE
Problems with Search after Upgrade to 1.1.4

### DISCUSSION THREAD

**Discussion by ciphermenial** [TandoorRecipes/recipes#1716] [Q&A] [answered]
I am seeing a strange issue with my search after an upgrade. It is failing to show anything on the **Search** page. All the recipes are accessible by clicking through other ways.
The strange part is that I set DEBUG=1 in the .env file and restarted and search works normally. Set DEBUG=0 and it is broken again.
Not sure what to try.

**Comment by smilerz** [comment]
see this issue: #1639

**Comment by ciphermenial** [comment, marked as answer]
I have updated and the issue is now gone.
